### ISSUE TITLE
`ipython==8.13.0` can be installed for python 3.8, but it is not designed to work with it.

### ISSUE THREAD

**Issue by anmyachev** [ipython/ipython#14053]
This is unexpected behavior. Can this be fixed?

**Comment by seuf** [comment]
+1 :
we have all our jobs failing on production suddenly :

```
ImportError:
IPython 8.13+ supports Python 3.9 and above, following NEP 29.
IPython 8.0-8.12 supports Python 3.8 and above, following NEP 29.
When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
Python 3.3 and 3.4 were supported up to IPython 6.x.
Python 3.5 was supported with IPython 7.0 to 7.9.
Python 3.6 was supported with IPython up to 7.16.
Python 3.7 was still supported with the 7.x branch.

See IPython `README.rst` file for more information:

    https://github.com/ipython/ipython/blob/main/README.rst
```

**Comment by basnijholt** [comment]
To make your CI jobs pass you can do something like: https://github.com/python-adaptive/adaptive/blob/466d5c35ab7de8da774540a023dee136ed3da46c/pyproject.toml#L33-L34

In `pyproject.toml`:
```
    "ipython; python_version > '3.8'",
    "ipython<8.13; python_version <= '3.8'", # because https://github.com/ipython/ipython/issues/14053
```

**Comment by Carreau** [comment]
I've yanked 8.13, and release 8.13.1 that should be properly tagged as 3.9+ only.

**Comment by Carreau** [comment]
And thanks @Kilo59 I'm not sure how I missed the python_requires in setup.cfg.

**Comment by Carreau** [comment]
Closing here. I should make 8.13.2 soon with a few other docs fixes related to 3.8

**Comment by Carreau** [comment]
Closing here. I should make 8.13.2 soon with a few other docs fixes related to 3.8
